Dockerfile.gpu

```diff
@@ -51,7 +53,6 @@ RUN cargo install --force --path ${HOME}/src/ds-srv/
 ENTRYPOINT sudo /sbin/ldconfig && nvidia-smi && ds-srv \
         -vvvv \
         --model $HOME/data/models/output_graph.pbmm \
-        --lm $HOME/data/models/lm.binary \
-        --trie $HOME/data/models/trie \
+        --scorer $HOME/data/models/kenlm \
```

Comment:
Does this path require the file extension? On Line 44 it's `${HOME}/data/models/kenlm.scorer` and here it's ` ${HOME}/data/models/kenlm`

Reply:
No that's a mistake...